bin/diagnostic.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+//We'll learn about require later in the course
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
```

Comment:
Super close! You'd just need some conditional check for the answer actually being your number of choice (which would need to be defined somewhere).